spec/BasicSpec.js

```diff
@@ -1140,6 +1140,26 @@ describe('HtmlWebpackPlugin', function () {
       /<script type="text\/javascript" src="common_bundle.js">.+<script type="text\/javascript" src="util_bundle.js">.+<script type="text\/javascript" src="index_bundle.js">/], null, done);
   });
 
+  it('should sort the chunks in the order provided in none mode', function (done) {
+    testHtmlPlugin({
+      entry: {
+        util: path.join(__dirname, 'fixtures/util.js'),
+        index: path.join(__dirname, 'fixtures/index.js')
+      },
+      output: {
+        path: OUTPUT_DIR,
+        filename: '[name]_bundle.js'
+      },
+      plugins: [
+        new HtmlWebpackPlugin({
+          chunks: ['util', 'index'],
```

Comment:
I should add that for option 2, it might be better to introduce a new sort mode (as suggested earlier in the thread), for example `chunkSortMode: 'explicit'`. This way `chunkSortMode: 'none'` still works the way it used to and when specifying `explicit`, then `chunks` option is required and used to do the sorting. I am happy to update the PR to work this way if you like.